x/evm/vm/geth/geth.go

```diff
@@ -0,0 +1,72 @@
+package geth
+
+import (
+	"math/big"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/vm"
+	"github.com/ethereum/go-ethereum/params"
+
+	evm "github.com/evmos/ethermint/x/evm/vm"
+)
+
+var (
+	_ evm.EVM         = (*EVM)(nil)
+	_ evm.Constructor = NewEVM
+)
+
+// EVM is the wrapper for the go-ethereum EVM.
+type EVM struct {
+	*vm.EVM
+}
+
+// NewEVM defines the constructor function for the go-ethereum EVM. It uses the
+// default precompiled contracts and the EVM from go-ethereum.
+func NewEVM(
+	blockCtx vm.BlockContext,
+	txCtx vm.TxContext,
+	stateDB vm.StateDB,
+	chainConfig *params.ChainConfig,
+	config vm.Config,
+	_ evm.PrecompiledContracts,
```

Comment:
Why is this required here? I mean I didn't see the custom contracts used anywhere.

Reply:
Because type Constructor func() in interface.go
If  _ evm.Constructor = NewEVM, then the variable needs to be there.
For example, NewEVM function can also be like this:

{
var c evm.Constructor
return c(blockCtx, txCtx, stateDB, chainConfig, config, nil)
}

Reply:
@CoderShiun Thanks for your explanation. Could I kindly bother you to wrap code sections in triple quotes (\```code section\```) for better readability? Also you can use single quotes (\`word\`) for highlighting something like `here`